CRM/Upgrade/3.2.alpha1.msg_template/message_templates/event_offline_receipt_text.tpl

```diff
@@ -20,9 +19,7 @@ a link to a web page where you can complete your registration.{/ts}
 {ts}Your registration has been submitted.{/ts}
 
 {if $isPrimary}
-{ts}Once your registration has been reviewed, you will receive
-an email with a link to a web page where you can complete the
-registration process.{/ts}
+{ts}Once your registration has been reviewed, you will receive an email with a link to a web page where you can complete the registration process.{/ts}
 
```

Comment:
Hmm, so what to do? {ts} is incompatible with line-breaks.
On a side-note, I was surprised to find the exact same template files over and over in our upgrade scripts. I fixed the same line-break issue 20 times or so. Is that really necessary?